docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -51,6 +51,38 @@ \subsection{Environment}
 
 \section{Goals}
 
+\subsection{Cater to Users' Interests}
+Allow for a wide variety of different categories and subcategories that a user may choose, in order that the system may provide a list of relevant recommendations.
+
+\subsection{Find Common Interests}
+Find content that meets some common criteria of all users in a group.
+
+\subsection{API Data Collection}
+Will obtain all necessary information on shows and movies through API services.
+
+\subsection{Group Compromise}
+Should there be no overlapping interests between two or more users in a group, the system will either provide show or movie recommendations that are of a similar 
+genre to what was selected (an ”inbetween” solution), use a tie-breaking mechanism (if one set of categories have more popularity in the group), or select options 
+from the non-overlapping categories.
+
+\subsection{Web App Compatibility}
+Will work on multiple web browsers, such as Google Chrome, Firefox, etc. on desktop devices.
+
+\subsection{Create Account and Login}
+User will be able to create an account with their saved preferences and account settings that are stored and will persist for future logins.
+
 \section{Stretch Goals}
 
+\subsection{Mobile Web Usability}
+System could be optimized for easier use on web browsers for mobile devices.
+
+\subsection{Mobile App}
+An app may made available on the Apple and Android stores for download.
+
+\subsection{Login with Other Accounts}
+Ability to log on to the system with another account, such as Facebook, Google, or Netflix.
+
+\subsection{Separate Dev and Prod Deployment}
+For the purpose of allowing developers to push up code without harming user experience if errors arise from it.
+
```

Comment:
Another Stretch Goal: link to the streaming service to be able to watch the recommendation on (but not actually watch the show through our website)

Reply:
+1, this is a solid idea